styles/onyx/elements/inputs/button/types/_icon.scss

```diff
@@ -23,20 +27,18 @@
   &.is-active,
   &.active,
   .open &.dropdown-toggle {
-    background-color: ice-adjust-color($button-default-bg-color, -0.5);
+    background: #DCDCDC;
+    z-index: 0;
   }
 
-  // Sometimes the icon buttons will have images rather than font awesome icons,
-  // so we need to use opacity for the hover effect
-  img {
-    transition: opacity $button-icon-transition-duration ease-out;
-    opacity: $button-icon-img-opacity;
-    height: $button-icon-img-size;
+  &:focus {
+    background: #DCDCDC;
+    z-index: 0;
```

Comment:
In the PR description you mentioned making sure the border is visible over sibling elements, but I think you meant the box shadow?

Pretty dangerous stuff :-/

Reply:
![image](https://user-images.githubusercontent.com/61712713/84515221-2beb3f80-ac9a-11ea-8ebe-ef1c234e0c86.png)

vs 

![image](https://user-images.githubusercontent.com/61712713/84515247-3574a780-ac9a-11ea-8f3e-40cd00f14170.png)

With a yellow border, the first one is with z-index set to 0 on the active element and the second is with z-index unset. Why is this dangerous? And is there a safer alternative?

Reply:
Oh I can just make it so that the border-left on the last-child is 0 lol

Reply:
@mixonic I paired with @twokul on this for a while and we were unable to come up with a solution that works for all the usages of `.button-icon` (There are cases with 3 adjacent buttons in a row). Wondering if @aschenoni has any ideas here, otherwise I think this is the best possible solution.

Reply:
`outline` comes to mind, I would expect that is above other things 🤔

Reply:
We ran into the same issue with outline :( 
![image](https://user-images.githubusercontent.com/61712713/84819269-b9ee5f80-afe5-11ea-8f13-4d0405f0b4f8.png)